client/db/bolt/db.go

```diff
@@ -104,6 +104,7 @@ var (
 	byteTrue               = encode.ByteTrue
 	backupDir              = "backup"
 	disabledRateSourceKey  = []byte("disabledRateSources")
+	walletStatusKey        = []byte("walletStatus")
```

Comment:
This is too vague of a name and string.  Why not `walletDisabledKey`?

Reply:
Does your suggestion apply to other use of `walletstatus`? e.g `togglewalletstatus` and `updateWalletStatus`?

Reply:
Just this probably.